Formula/o/ouch.rb

```diff
@@ -42,13 +46,13 @@ def install
     (testpath/"file1").write "Hello"
     (testpath/"file2").write "World!"
 
-    %w[tar zip tar.bz2 tar.gz tar.xz tar.zst].each do |format|
+    %w[tar zip 7z tar.bz2 tar.bz3 tar.lz4 tar.gz tar.xz tar.zst tar.sz tar.br].each do |format|
       system bin/"ouch", "compress", "file1", "file2", "archive.#{format}"
       assert_path_exists testpath/"archive.#{format}"
 
       system bin/"ouch", "decompress", "-y", "archive.#{format}", "--dir", testpath/format
-      assert_equal "Hello", (testpath/format/"archive/file1").read
-      assert_equal "World!", (testpath/format/"archive/file2").read
+      assert_equal "Hello", (testpath/format/"file1").read
+      assert_equal "World!", (testpath/format/"file2").read
```

Comment:
Is this just reading the files that were written before running `ouch compress` and `ouch decompress`?